tests/testthat/files/source_values.R

```diff
@@ -0,0 +1,13 @@
+#* @get /a
```

Comment:
Maybe replace this with a test that counts how many times the code has been evaluated.

```R
#* @get /count
{
  if (!exists("count")) {
    count <- 0
  }
  count <- count + 1
  function() {
    count
  }
}
```